src/lib/signer.ts

```diff
@@ -13,21 +14,35 @@ export const getSigner = ({
   signerId: string;
   keysPath: string;
   lcd: LCDClient;
-}): Wallet => {
+}): Promise<Wallet> => {
   const localterra = new LocalTerra();
   if (
     network === 'localterra'
     && Object.prototype.hasOwnProperty.call(localterra.wallets, signerId)
   ) {
     cli.log(`using pre-baked '${signerId}' wallet on localterra as signer`);
+
     // @ts-ignore
-    return localterra.wallets[signerId];
+    const signer = localterra.wallets[signerId];
+
+    // Attempt to make a request to LocalTerra.
+    // Alert user if LocalTerra request fails.
+    try {
+      await signer.sequence();
+      return new Promise(signer);
```

Comment:
```suggestion
      return signer;
```

For async functions any return is automagically wrapped in a promise.